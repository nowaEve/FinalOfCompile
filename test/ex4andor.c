void main() { 
    int x = 4;
    int y = 5;
    if((x<y)&&(x!=0)){
        print x;
    }
    else{
        print y;
    }
    if(x==4||y==5){
        print x;
    }
    else{
        print y;
    }
}