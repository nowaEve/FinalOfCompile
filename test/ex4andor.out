24 19 0 5 25 13 0 4 12 13 0 1 1 0 5 12 13 11 13 0 1 1 11 7 17 37 13 11 17 37 13 11 22 15 -1 16 45 13 0 1 1 11 22 15 -1 13 11 0 4 6 18 62 13 0 1 1 11 0 5 6 17 67 13 11 22 21 2 13 0 1 1 11 22 21 2